docs/options.md

```diff
@@ -319,6 +319,100 @@ Path to uncrustify config file. i.e. uncrustify.cfg (Supported by Uncrustify)
 }
 ```
 
+####  [Bash](#bash) 
+
+**Supported Beautifiers**:  [`beautysh`](#beautysh) 
+
+| Option | beautysh |
+| --- | --- |
+| `disabled` | :white_check_mark: |
+| `default_beautifier` | :white_check_mark: |
+| `beautify_on_save` | :white_check_mark: |
+| `configPath` | :white_check_mark: |
```

Comment:
@quilicicf this should also be removed. Remove `configPath` from language and beautifier and re-generate documentation. Thanks!